### PR TITLE
fix flake8 warnings

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: stock_picking_webkit/__init__.py [hunk 1]
```diff
@@ -19,4 +19,4 @@
 #
 ##############################################################################
 
-import report
+from . import report  # noqa
```
Comment (bwrsandman) [inline stock_picking_webkit/__init__.py:22]: https://github.com/OCA/stock-logistics-reporting/pull/3#issuecomment-47952750

File: stock_picking_webkit/report/__init__.py [hunk 2]
```diff
@@ -18,4 +18,4 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-import stock_report
+from . import stock_report  # noqa
```
Comment (bwrsandman) [inline stock_picking_webkit/report/__init__.py:21]: https://github.com/OCA/stock-logistics-reporting/pull/3#issuecomment-47952750

**Comment by yvaucher** [comment]
@gurneyalex closing as 6.1 is green

**Comment by pedrobaeza** [comment]
I would suspect about current green... Can we force a rebuild?

**Pull request closed without merging**